gpu.Dockerfile

```diff
@@ -59,7 +59,7 @@ ENV CUDA_HOME=/usr/local/cuda
 ADD ldpaths $R_HOME/etc/ldpaths
 
 # Install tensorflow with GPU support
-RUN R -e 'keras::install_keras(tensorflow = "2.3-gpu")' && \
+RUN R -e 'keras::install_keras(tensorflow = "2.4-gpu")' && \
```

Comment:
No CUDA upgrade necessary? Do you have a test exercising TF + GPU?

Reply:
I see this one: https://github.com/Kaggle/docker-rstats/blob/master/tests/test_tensorflow.R#L7

Reply:
I was also surprised it was that easy.